Latest Windows attempt do build the repo: running `restore.cmd` from root I get this:
```
...\aspnetcore\src\Servers\IIS\build\Build.Common.Settings(12,3): error MSB4019: The imported project "...\
aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Microsoft\VC\v160\Microsoft.Cpp.Default.props" was not found. Conf
irm that the expression in the Import declaration "...\aspnetcore\.tools\msbuild\16.5.0-alpha\tools\MSBuild\Micro
soft\VC\v160\\Microsoft.Cpp.Default.props" is correct, and that the file exists on disk. [...\aspnetcore\src\Serv
ers\IIS\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj]
```

No idea what I'm doing wrong :-(

--- a/src/Components/test/E2ETest/Tests/BindTest.cs
+++ b/src/Components/test/E2ETest/Tests/BindTest.cs
@@ -808,6 +808,7 @@ public class BindTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>
     // For date comparisons, we parse (non-formatted) values to compare them. Client-side and server-side
     // Blazor have different formatting behaviour by default.
     [Fact]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/41425")]
     public void CanBindTextboxNullableDateTimeOffset()
     {
         var target = Browser.Exists(By.Id("textbox-nullable-datetimeoffset"));
@@ -898,6 +899,7 @@ public class BindTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>
     // For date comparisons, we parse (non-formatted) values to compare them. Client-side and server-side
     // Blazor have different formatting behaviour by default.
     [Fact]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/41425")]
     public void CanBindTextboxTimeOnly()
     {
         var target = Browser.Exists(By.Id("textbox-timeonly"));

